doc/ref_model/chapters/chapter04.rst

```diff
@@ -77,6 +77,8 @@ e.cap.013 SR-IOV over PCI-PT              Yes/No Traditional SR-IOV. These Capab
 e.cap.014 GPU/NPU                         Yes/No Hardware coprocessor. These Capabilities generally require hardware-dependent drivers be injected into workloads
 e.cap.015 SmartNIC                        Yes/No Network Acceleration
 e.cap.016 FPGA/other Acceleration HW      Yes/No These Capabilities generally require hardware-dependent drivers be injected into workloads
+e.cap.023 Huge pages                      Yes/No Indicates if the Cloud Infrastructure supports huge pages
```

Comment:
The table below in "Profiles Specifications & Capability Mapping" (I didn't see a table number) still lists Huge page as i.cap.18.

Reply:
I was not talking about table 4-7 but about the table without table number in clause "Profiles Specifications & Capability Mapping" further below.

Reply:
It is in line 352, which I cannot comment, maybe because it is below the PR content.

Reply:
Thanks, @pgoyal01